DMTR-451.tex

```diff
@@ -46,24 +46,16 @@
-
-    \item The Alert Production pipeline switched to the new CalibrateImageTask instead of the former CharacterizeImageTask and CalibrateTask.
-
-    \item The synthetic source injection package is now included in lsst\_distrib. This update is accompanied by documentation that provides detailed instructions on how to use it and an FAQ.
+    \item Compensated tophats for photoCalib (DM-38632), used in CalibrateImageTask (DM-44908).
+    \item DM-44130 change default S/N cut for PSF star selection from 20 to 50.
+    \item DM-44305 Implement Gaussian Process interpolation of bad pixels.
 \end{itemize}
 
-Photometry and astrometry metrics reported here were calculated using the \href{https://github.com/lsst/analysis_tools}{analysis\_tools} package, which is part of the standard pipeline distribution. The \texttt{analysis\_tools} package builds on and supersedes \href{https://github.com/lsst/faro}{faro} (\citeds{DMTN-211}), which has been used for the past few years. The calculation of most metrics reported in this Report is unchanged between the two packages, though minor differences are to be expected because of fundamental differences in the underlying data-handling frameworks. As discussed in the next section, the photometric outlier fraction PF1 has been updated relative to previous pipeline release versions in order to fix a mistake in how the calculation was implemented. We are actively working on a revised definition of the residual ellipticity correlation metrics TE1 and TE2, to be implemented in \texttt{analysis\_tools}. Because this work is still in progress, the ellipticity metrics reported here are calculated by \texttt{faro} in the same manner as previous versions of the Characterization Metric Report.
+Photometry and astrometry metrics reported here were calculated using the \href{https://github.com/lsst/analysis_tools}{analysis\_tools} package, which is part of the standard pipeline distribution. The \texttt{analysis\_tools} package builds on and supersedes \href{https://github.com/lsst/faro}{faro} (\citeds{DMTN-211}), which has been used for the past few years. The calculation of most metrics reported in this Report is unchanged between the two packages, though minor differences are to be expected because of fundamental differences in the underlying data-handling frameworks. We are actively working on a revised definition of the residual ellipticity correlation metrics TE1 and TE2, to be implemented in \texttt{analysis\_tools}. Because this work is still in progress, the ellipticity metrics reported here are calculated by \texttt{faro} in the same manner as previous versions of the Characterization Metric Report.
```

Comment:
"though minor differences are to be expected because of fundamental differences in the underlying data-handling frameworks." -- Written like this, it seems to imply that the frameworks themselves influence the results, which should not be the case